sklearn/decomposition/kernel_pca.py

```diff
@@ -34,38 +34,38 @@ class KernelPCA(BaseEstimator, TransformerMixin):
     degree : int, default=3
         Degree for poly kernels. Ignored by other kernels.
 
-    gamma : float, optional
-        Kernel coefficient for rbf and poly kernels. Default: 1/n_features.
-        Ignored by other kernels.
+    gamma : float, default=1/n_features
+        Kernel coefficient for rbf and poly kernels. Ignored by other
+        kernels.
 
-    coef0 : float, optional
+    coef0 : float, default=1
         Independent term in poly and sigmoid kernels.
         Ignored by other kernels.
 
-    kernel_params : mapping of string to any, optional
+    kernel_params : mapping of string to any, default=None
         Parameters (keyword arguments) and values for kernel passed as
         callable object. Ignored by other kernels.
 
-    alpha : int, default=1.0
+    alpha: int, default=1.0
         Hyperparameter of the ridge regression that learns the
         inverse transform (when fit_inverse_transform=True).
 
-    fit_inverse_transform : bool, default=False
+    fit_inverse_transform: bool, default=False
         Learn the inverse transform for non-precomputed kernels.
         (i.e. learn to find the pre-image of a point)
 
-    eigen_solver : string ['auto'|'dense'|'arpack']
-        Select eigensolver to use. If n_components is much less than
+    eigen_solver: string ['auto'|'dense'|'arpack'], default='auto'
+        Select eigensolver to use.  If n_components is much less than
         the number of training samples, arpack may be more efficient
```

Comment:
I think it seems okay as is. "arpack" is referring to the string, while arpack is referring to the library.